src/test/java/copy_credentials/CopyCredentialsInteractorTest.java

```diff
@@ -47,4 +53,52 @@ public void displayCopyMessage(String message) {
         copyCredentialsInputBoundary.copyPassword(passwordInputData);
         assertEquals((Toolkit.getDefaultToolkit().getSystemClipboard().getData(DataFlavor.stringFlavor)), testPassword);
     }
+
+
+    @Test
+    public void testClearClipBoard() throws IOException, UnsupportedFlavorException, InterruptedException {
+        PasswordInputData passwordInputData = new PasswordInputData(testPassword);
+        CopyCredenentialsOutputBoundary copyCredenentialsOutputBoundary = new CopyCredenentialsOutputBoundary() {
+            @Override
+            public void displayCopyMessage(String message) {
+
+            }
+
+        };
+        CopyCredentialsInputBoundary copyCredentialsInputBoundary = new CopyCredentialsInteractor(copyCredenentialsOutputBoundary);
+        // Set the clipboard to the test password
+        Toolkit.getDefaultToolkit().getSystemClipboard().setContents(new StringSelection(testPassword), null);
+
+        // Verify the clipboard was set correctly
+        assertEquals(testPassword, Toolkit.getDefaultToolkit().getSystemClipboard().getData(DataFlavor.stringFlavor));
+        copyCredentialsInputBoundary.clearClipboard();
+        final CountDownLatch latch = new CountDownLatch(1);
+
+        final Timer timer = new Timer();
+        final TimerTask timerTask = new TimerTask() {
+            @Override
+            public void run() {
+                try {
+                    // Perform the assertion inside the timer task
+                    assertEquals("",
+                            Toolkit.getDefaultToolkit().getSystemClipboard().getData(DataFlavor.stringFlavor)
+                    );
+                } catch (UnsupportedFlavorException | IOException e) {
```

Comment:
we don't checkstyle on *Test.java files but if it did, you should rename "e" to "exception" here